hadoop-yarn-project/hadoop-yarn/hadoop-yarn-csi/pom.xml

```diff
@@ -150,12 +150,12 @@
                 <artifactId>protobuf-maven-plugin</artifactId>
                 <version>${protobuf-maven-plugin.version}</version>
                 <configuration>
-                    <protocArtifact>com.google.protobuf:protoc:3.6.1:exe:${os.detected.classifier}</protocArtifact>
                     <pluginId>grpc-java</pluginId>
                     <pluginArtifact>io.grpc:protoc-gen-grpc-java:1.15.1:exe:${os.detected.classifier}</pluginArtifact>
                 </configuration>
                 <executions>
                     <execution>
+                        <id>src-compile-protoc</id>
```

Comment:
This will change the default output directory "target/generate-sources/protobuf/java" to "target/generate-sources/java"
So, next plugin "build-helper-maven-plugin" in this project may fail.
yarn-csi project was not affected by the protobuf upgrade. So better to keep it as is.

Reply:
I think we'd better align this. Otherwise it will be confusing to developpers, as we define the plugin in the parent pom and reference it here, but then we just skip all the exections from the parent pom and define a new compile execution.

Anyway, we can keep it as is in this patch, and file a follow-on issue to address it.

Reply:
Yes, follow on would be better.